### ISSUE TITLE
helm configure non existing namespace does nothing (and no error message)

### ISSUE THREAD

**Issue by ebini** [hashicorp/terraform-provider-helm#427]
### Terraform Version
Terraform v0.12.20
+ provider.helm v1.0.0

### Affected Resource(s)
- helm_release

### Terraform Configuration Files
```hcl
resource "helm_release" "loki" {
  name       = "lokiterraform"
  repository = data.helm_repository.stable.metadata[0].name
  chart      = "loki-stack"
  namespace  = "test"
}
```

### Expected Behavior
create namespace or exit with error

### Actual Behavior
exits with no error, but nothing has been created

### Steps to Reproduce
1. `terraform apply`

**Comment by zdevi** [comment]
Hi, I have the exact same problem, if namespace doesn't exist, no ressources are created, but terraform ends saying all is OK.
Working good if i previously create the namespace.

**Comment by aaronmell** [comment]
Its likely that helm doesn't return an error. Can you try creating a chart with helm directly and see if you get an error message?

**Comment by dawidmalina** [comment]
It will return error message like this:
```
Error: create: failed to create: namespaces "test" not found
```

Looks like this functionality (namespace creation) was remove in helm3 - https://github.com/helm/helm/issues/6794

With the new helm release `--create-namespace` option will be added - https://github.com/helm/helm/pull/7648

**Comment by jrhouston** [comment]
This was fixed in 1.1.0.
